stock_inventory_lockdown/models/stock_quant.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# © 2016 Numérigraphe SARL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, _
+from openerp.exceptions import ValidationError
+
+
+class StockQuant(models.Model):
+    _inherit = 'stock.quant'
+
+    @api.multi
+    def write(self, vals):
+        """Check that the location is not locked by an open inventory.
+        Check both the location as it was (source) and the location as
+        it will be (destination).
+        We verify the locations even if they are unchanged, because changing
+        ie. the quantity is not acceptable either.
+        @raise ValidationError if they are.
+        """
+        if not self.env.context.get('bypass_lockdown', False):
+            # Find the locked locations
+            locked_location_ids = self.env[
```

Comment:
@mreficent This code should be also modified to take advantage of the new paramater on `_get_locations_open_inventories`

```python
if 'location_id' in vals.keys():
    locked_location_ids = self.env['stock.inventory']._get_locations_open_inventories(
        self.env['stock.location'].browse(vals['location_id']) + self.mapped('location_id')
    )
   if locked_location_ids:
        raise ...
```